frontend/app/src/components/home/AggregateCommonEvents.svelte

```diff
@@ -85,7 +85,7 @@
                 class="deleted"
                 title={$_("expandDeletedMessages")}
                 bind:this={deletedMessagesElement}
-                data-index={messagesDeleted.join(" ")}
+                data-index={Math.max(...messagesDeleted)}
```

Comment:
Tiny bit worried about whether that has any implications for [this](https://github.com/open-chat-labs/open-chat/blob/master/frontend/app/src/components/home/ChatEventList.svelte#L376)